.github/workflows/ci.yml

```diff
@@ -67,6 +67,7 @@ jobs:
         # Run the end-to-end tests against the code in this branch specifically:
         npm install ../../
         cd ../..
+      if: matrix.node-version != '14.x' && matrix.node-version != '12.x' && github.actor != 'dependabot[bot]'
```

Comment:
Just a minor nit, but rather than these explicit references to version numbers, can we define a new 'node-version-latest' matrix param:
```yaml
strategy:
      matrix:
        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
        os: [ubuntu-20.04, windows-2019, macos-10.15]
        node-version-latest: 16.x
        node-version: [node-version-latest, 14.x, 12.x]
```
...and then this line becomes:
```yaml
if: matrix.node-version = matrix.node-version-latest && github.actor != 'dependabot[bot]'
```